aggregation_server/config.toml

```diff
@@ -0,0 +1,4 @@
+port = "$PORT"
+max_workers = "$MAX_WORKERS"
+timeout = "$TIMEOUT"
```

Comment:
Could you be a little more specific on the name? `timeout` is very generic... 

Same for max_workers, maybe `max_grpc_connections` ? Is it actually the max grpc connections?

Reply:
maxWaitForAggregationInputs